spec/services/diversities/disclosures/update_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "rails_helper"
+
+module Diversities
+  module Disclosures
+    describe Update do
+      describe ".call" do
+        let(:trainee) { create(:trainee, diversity_disclosure: nil) }
+        let(:service) { described_class.new(trainee: trainee, attributes: attributes) }
+
+        context "when disclosure attribute is valid" do
+          let(:attributes) { { diversity_disclosure: "yes" } }
+
+          before do
+            service.call
+            trainee.reload
+          end
```

Comment:
can probably move it to the top

```ruby
describe Update do
  describe ".call" do
    let(:trainee) { create(:trainee, diversity_disclosure: nil) }
    let(:service) { described_class.new(trainee: trainee, attributes: attributes) }
    before do
        service.call
        trainee.reload
    end

```

Reply:
done https://github.com/DFE-Digital/register-trainee-teachers/pull/104/commits/8150d6251de7a79e8a063c63b2656f5babac358a